.github/workflows/tests.yml

```
@@ -28,6 +28,12 @@ jobs:
       os: ubuntu-latest
       python-version: '3.10'
       tox-env: py310
+  linux-py38-oldestdeps:
+    uses: ./.github/workflows/python-test-workflow.yml
+    with:
+      os: ubuntu-latest
+      python-version: '3.10'
```

This should be the same as the `py38` tox env?
```suggestion
      python-version: '3.8'
```